BackEndApp/v1/utils.py

```diff
@@ -22,7 +24,10 @@ def get_augument_params():
 def copy_file(dest, filename="test.py"):
     if not filename.endswith(".py"):
         filename += ".py"
-    os.system('cp -f {} "{}"/{}'.format(filename, dest, filename))
+    fsource = os.path.join(os.getcwd(), filename)
```

Comment:
Just a small change. Set the source filename to something unique like `{filename}_{timestamp}` so that if there are multiple simultaneous requests by different users the contents don't get overwritten.

Reply:
You'll have to update `generate()` in `BackEndApp/v1/views.py`

Reply:
I believe the source_filename is a part of the DLMML package, so I won't be able to update it.

Reply:
You both are right :)
We can do something better tho (guess?), so for now, I believe we can let this slide.